misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/ComparableAssertionsSpec.kt

```diff
@@ -99,6 +102,24 @@ abstract class ComparableAssertionsSpec(
                 fluent.isGreaterOrEqualsFun(9)
             }
         }
+
+        describe("${isEqualComparingTo.name} ...") {
+            val isEqualComparingToFun = isEqualComparingTo.lambda
+
+            it("... 11 throws an AssertionError containing ${DescriptionComparableAssertion::class.simpleName}.${DescriptionComparableAssertion.IS_EQUAL} and `: 11`") {
+                expect {
+                    fluent.isEqualComparingToFun(11)
+                }.toThrow<AssertionError> { messageContains("$isEqualComparingToDescr: 11") }
+            }
+            it("... 10 does not throw") {
+                fluent.isEqualComparingToFun(10)
+            }
+            it("... 9 throws an AssertionError containing ${DescriptionComparableAssertion::class.simpleName}.${DescriptionComparableAssertion.IS_EQUAL} and `: 9`") {
+                expect {
+                    fluent.isEqualComparingToFun(9)
+                }.toThrow<AssertionError> { messageContains("$isEqualComparingToDescr: 9")}
+            }
+        }
```

Comment:
Can you please also add a test-case where the classes' equal behaves differently than compareTo. You can use an own implementation. For instance:
```
data class DiffEqualsCompare(val s: String) : Comparable<DiffEqualsCompare>{
    override fun compareTo(other: DiffEqualsCompare): Int {
        other.s.equals("hello")
    }
}
```
and then 
```
expect(DiffEqualsCompare("s")).toBe(DiffEqualsCompare("s")
```
should pass and
```
expect(DiffEqualsCompare("s")).isEqualComparingTo(DiffEqualsCompare("s")
```
should fail and
```
expect(DiffEqualsCompare("s")).isEqualComparingTo(DiffEqualsCompare("hello")
```
should pass

Reply:
I'm imagining the first part:

`expect(DiffEqualsCompare("s")).toBe(DiffEqualsCompare("s")`

wouldn't make sense in a test case by itself, right? (since it's not actually testing this new method)

Should I put all three of these into one test, or should I put them separately?

Reply:
Agreed on the first point and up to you if you put it in one or two test cases